telemetry/data_cluster_hardware_test.go

```diff
@@ -20,39 +20,81 @@ import (
 )
 
 func TestNormalizeDiskName(t *testing.T) {
-	t.Parallel()
-	require.Equal(t, "sdb", normalizeDiskName("/dev/sdb"))
-	require.Equal(t, "sda", normalizeDiskName("sda"))
+	tests := []struct {
+		diskName string
+		expected string
+	}{
+		{"/dev/sdb", "sdb"},
+		{"sda", "sda"},
+	}
+
+	for _, test := range tests {
+		t.Run(test.diskName, func(t *testing.T) {
+			t.Parallel()
```

Comment:
I think a normal loop with comments is OK. Sub tests are fine, too. `Parallel` seems overkill, maybe call it outside the loop.

Reply:
OK.

Reply:
updated.